clusterloader2/testing/l4lb/service.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{.Name}}
+  labels:
+    test: l4lb-load
+  annotations:
+    cloud.google.com/load-balancer-type: {{.LoadBalancerType}}
+spec:
+  externalTrafficPolicy: {{.ExternalTrafficPolicy}}
+  type: LoadBalancer
+  selector:
+    name: {{.DeploymentBaseName}}-{{.Index}}
+  ports:
+  - port: 8080
+    targetPort: 80
```

Comment:
Shouldn't it match `containerPort: 8080` ?

Reply:
good catch

Reply:
port:8080 and targetPort: 80 should be ok as well, since the ServiceCreationLatency measurement will reach out to VIP:8080 in that case - https://github.com/kubernetes/perf-tests/blob/bb0129087ee1ac79d1800cad913e7f8a252e059b/clusterloader2/pkg/measurement/common/service_creation_latency.go#L340